### PR TITLE
Archive ops 53

### PR DESCRIPTION
**Pull request by TBRC-Travis**
Review updated code

### CODE REVIEW

File: scripts/QA/check-BUDA-status.sh [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+  ridTuples=($(curl -s "http://purl.bdrc.io/query/table/Work_ImgList?R_RES=bdr:${w}&format=csv&profile=simple&pageSize=500"))
+
+  igList=()
+
+  # loop through the tuples, extract each IG ID, and add to image group array (igList)
+  for tuple in "${ridTuples[@]}"; do
+    # extract IG ID from tuple
+    ig=$(echo $tuple | cut -d"," -f3 | tr -d '\"')
+     
+    # make sure we don't add the grId column header to the array
```
Comment (TBRC-JimK) [inline scripts/QA/check-BUDA-status.sh:69]: Nice doc

File: scripts/QA/check-BUDA-status.sh [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+  # Check each image group ID
+  for ig in "${igList[@]}"; do
+    echo $ig
+
+    # Ping BUDA dev to determines if images are loading or not
+    itsAlive=$(curl -o /dev/null -s  -w "%{http_code}\n" https://iiifpres-dev.bdrc.io/v:bdr:${ig}/manifest)
+
+    # If we get a 200 response we're golden -- the images load on BUDA
+    if [[ $itsAlive == "200" ]]; then
+      echo "$w-$ig" >> $logDir/${workList%.*}-${xdate}-${xtime}-IGs-IMAGES-FOUND.txt
```
Comment (TBRC-JimK) [inline scripts/QA/check-BUDA-status.sh:84]: No foul, but better to initialize these log file names up top. Shorter cleaner lines of code
 ```
prefix=$logDir/${workList%.*}-${xdate}-${xtime}-IGs-IMAGES-
goodListFile=${prefix}FOUND.txt
badListFile=$logDir/${workList%.*}-${xdate}-${xtime}-NOTFOUND.txt
```

and so on.
That way, the main code gets
`echo "$w-$ig" >> $goodList`

File: scripts/QA/check-BUDA-status.sh [hunk 3]
```diff
@@ -0,0 +1,95 @@
+#!/bin/sh
+
```
Comment (TBRC-JimK) [inline scripts/QA/check-BUDA-status.sh:2]: Needs an explain of what "checking BUDA status" means. What this really checks is if the BUDA IIIF viewer can display the work.

**Comment by TBRC-Travis** [comment]
Beautified and merged.  Good tips.  Thanks!

https://github.com/buda-base/archive-ops/blob/master/scripts/QA/check-BUDA-status.sh

**Pull request merged**